tree.go

```diff
@@ -99,6 +110,7 @@ type Node struct {
 	Focus              []NodeID         `json:"focus"`
 	Nodes              []*Node          `json:"nodes"`
 	FloatingNodes      []*Node          `json:"floating_nodes"`
+	Floating           string           `json:"floating"`
```

Comment:
Shouldn’t this field be of type FloatingType?

Reply:
fixed